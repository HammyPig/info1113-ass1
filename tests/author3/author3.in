AUTHOr Virgil
ADD COLLECTION books.csv
auTHOR Virgil
author W.E. Johns
AUTHOR J.K. Rowling