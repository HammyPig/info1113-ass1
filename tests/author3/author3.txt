recognises multi word authors, and can output multiple books per author