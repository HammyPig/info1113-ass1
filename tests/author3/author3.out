user: No books in system.

user: 17 books successfully added.

user: Aeneid (Virgil)

user: Biggles in the Battle (W.E. Johns)

user: Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: Ending Library process.
